Gemfile

```diff
@@ -7,6 +7,7 @@ ruby File.read('.ruby-version')
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'coffee-rails'
+gem 'dogstatsd-ruby', require: 'datadog/statsd' # send metrics to datadog agent
```

Comment:
I thought there might be a peer dependency on `ddtrace` but while testing locally I was able to report metrics just with `dogstatsd-ruby`.

Reply:
Can't believe that Horizon never had `ddtrace`! :face-palm: